### PR TITLE
fix:  prepare sql use single frontConnection not multiple frontConnection inner 594

### PR DESCRIPTION
**Pull request by ylinzhu**
Reason:  
  BUG 
Type:  
  BUG
Influences：  
    conflicts:src/main/java/com/actiontech/dble/services/rwsplit/RWSplitQueryHandler.java
    fix:  prepare sql use single frontConnection not multiple frontConnection inner 594

### CODE REVIEW

File: src/main/java/com/actiontech/dble/rwsplit/RWSplitNonBlockingSession.java [hunk 1]
```diff
@@ -32,6 +41,28 @@ public void execute(Boolean master, Callback callback) {
         execute(master, null, callback);
     }
 
+    public void execute(Boolean master, Callback callback, String sql) {
+        try {
+            SQLStatement statement = parseSQL(sql);
+            if (statement instanceof MySqlPrepareStatement) {
+                String simpleName = ((MySqlPrepareStatement) statement).getName().getSimpleName();
+                nameSet.add(simpleName);
+                rwSplitService.setTxStart(true);
```
Comment (wenyh1) [inline src/main/java/com/actiontech/dble/rwsplit/RWSplitNonBlockingSession.java:50]: application point？
prepare has no practical relationship with tx; (doing so may lead to other aspects of influence)
If it's for some purpose, try something else.

**Comment by LUAgam** [comment]
Does [DROP PREPARE](https://dev.mysql.com/doc/refman/8.0/en/deallocate-prepare.html) need to be implemented?

**Comment by ylinzhu** [comment]
druid not support  DROP PREPARE

**Review by wenyh1 — DISMISSED** [review]
wait for the follow-up support DROP PREPARE, need review.

**Pull request closed without merging**